_data/toc.yaml

```diff
@@ -139,8 +141,8 @@ guides:
       path: /get-started/part4/
     - title: "Part 5: Sharing Images on Docker Hub"
       path: /get-started/part5/
-  - path: /engine/docker-overview/
-    title: Docker overview
+  - path: /get-started/resources/
+    title: "Educational Resources"
```

Comment:
Should this be Educational resources to be consistent with other TOC entries?

Reply:
prolly, made the update.